scripts/asm-installer/cloudbuild-master.yaml

```diff
@@ -78,7 +78,7 @@ steps:
     args:
       - '-c'
       - >
-        ./release_asm_installer
+        ./../release-asm/release_asm_installer
```

Comment:
You can directly use `../release-asm/release_asm_installer`

Reply:
Done